### DISCUSSION TITLE
Board Part # STM32L433RCIx still not visible in the Arduino tools menu.

### DISCUSSION THREAD

**Discussion by Siddique1** [stm32duino/Arduino_Core_STM32#2504] [Q&A] [answered]
**Please, before reporting any issue**
- Ensured the latest Arduino_Core_STM32 version = 2.8.1, see the snapshot.
- Ensured not discussed on stm32duino forum

I can't see STM32L433RCIx in the Board Part # option. See the picture
![STM32L433RCIx_VariantIssue](https://github.com/user-attachments/assets/b3605526-3e4d-4d44-908c-f942b10a63ab)

I am using a custom STM32L433RCI UFBGA packaged custom BGA board. I tried with STM32L452RCIx and it worked. Folder structure for Variants for STM32L452RCIx and STM32L433RCIx is same.

**Comment by fpistm** [comment, marked as answer]
Hi @Siddique1 
don't know why you don't see it as it is present (tested on Windows and Linux):
![image](https://github.com/user-attachments/assets/7c99493b-be60-4bf3-9dd0-a56089f3697c)
And is well declared:
https://github.com/stm32duino/Arduino_Core_STM32/blob/0935852a6ba127921f9f0eb8ab295747daf14e26/boards.txt#L11037-L11044

You can try to delete Arduino IDE cache or uninstall/reinstall the core.
Anyway nothing I can do here.

**Reply by Siddique1** [reply]
Hi, Thanks @fpistm for sharing. I have updated the STM32_Arduino_core and it didn't work as mentioned. Now uninstalled and then reinstalled latest version and it worked.

**Reply by fpistm** [reply]
Thanks for the feedback, probably an issue with the Arduino IDE cache. Somme issue already raised on Arduino side ...
